src/main/java/seedu/address/model/assignment/DueDate.java

```diff
@@ -9,13 +10,14 @@
 
 public class DueDate {
     public static final String MESSAGE_CONSTRAINTS = "Due dates should be in a format dd/MM/yyyy HHmm";
+    public static final String DATE_VALIDATION_REGEX =
+            "^([1-9]|[0-2][0-9]|(3)[0-1])(/)([1-9]|((0)[0-9])|((1)[0-2]))(/)\\d{4}$";
```

Comment:
Perhaps you can consider validation of dates such as ```30/02/2019```?

Reply:
The validation of dates already take into account of that test case, I will update the test file to include that